src/routes/ethereum-navigator/+page.svelte

```
@@ -0,0 +1,194 @@
+        searchTimeout = setTimeout(() => {
+            const searchLower = query.toLowerCase();
+            filteredPosts = posts?.filter(post => 
+                !query || 
+                post.title.toLowerCase().includes(searchLower) ||
+                post.preview_text.toLowerCase().includes(searchLower)
+            ) ?? [];
+        }, 300);
+    }
+
+    // Reactive statement to trigger search
+    $: searchQuery && handleSearch(searchQuery);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)